### ISSUE TITLE
NormalFloatHyperparameter loses bounds when written to JSON

### ISSUE THREAD

**Issue by LGro** [automl/ConfigSpace#276]
Serializing the following space with a normally distributed float parameter
```python
from ConfigSpace import ConfigurationSpace, Float, Normal
from ConfigSpace.read_and_write import json

cs = ConfigurationSpace(
    space={
        "p": Float(
            "p",
            bounds=(1e-5, 1e-1),
            default=0.01,
            log=True,
            distribution=Normal(1.0, 0.6),
        ),
    },
)

print(json.write(cs))
```

produces the output
```json
{
  "hyperparameters": [
    {
      "name": "p",
      "type": "normal_float",
      "log": true,
      "mu": 1.0,
      "sigma": 0.6,
      "default": 0.01
    }
  ],
  "conditions": [],
  "forbiddens": [],
  "python_module_version": "0.6.0",
  "json_format_version": 0.4
}
```

which I expected to contain the bounds as well, but they seem to get lost during JSON serialization.

A test case that I'd expect to pass and cover this issue could look like
```python
cs1 = ConfigurationSpace(
    space={
        "p": Float(
            "p",
            bounds=(1e-5, 1e-1),
            default=0.01,
            log=True,
            distribution=Normal(1.0, 0.6),
        ),
    },
)
cs2 = json.read(json.write(cs1))
assert cs1 == cs2
```

**Comment by eddiebergman** [comment]
Heyo,

Thanks for pointing this out! I'll try get something done for it before Christmas holibreaks :)

**Comment by mfeurer** [comment]
@Neonkraft can this be closed?

**Comment by Neonkraft** [comment]
> @Neonkraft can this be closed?

Yes. Resolved in #308
